### PR TITLE
[ST-348] Add initial sequence recorder docs

### PR DESCRIPTION
**Pull request by davidgm0**
### Added

Sequence recorder. Installation and links aren't there yet, but I would like to have this page before submitting the extension

| Page | Deploy Preview | Comment |
| ----- | ----- | ----- |
| Name | Link | Comment |

### Updated

| Page | Deploy Preview | Comment |
| ----- | ----- | ----- |
| Name | Link | Comment |

## Preview This Change

To see how this change looks in production, scroll down to **Deploy Preview**. Select the link that looks like `https://deploy-preview-<num>--cobalt-docs.netlify.app/`

## Variables

Help us support a “Write once, publish everywhere” single source of truth. If you see a line that looks like:

`{{% asset-categories %}}`

You’ve found a [shortcode](https://gohugo.io/content-management/shortcodes/) that we include in multiple documents.

You’ll find the content of the shortcode in the following directory:

https://github.com/cobalthq/cobalt-product-public-docs/tree/main/layouts/shortcodes

That shortcode has the same base name as what you see in the PR, such as `asset-categories.html`.

## Checklist for PR Author

[ ] Did you check for broken links and alt text?

Be sure to check for broken links and Alt Text issues. We have a partially automated process,
as described in this section of our repository README:
[Test Links and Alt Attributes](https://github.com/cobalthq/cobalt-product-public-docs/blob/main/README.md#test-links-and-alt-attributes).

### CODE REVIEW

File: content/en/Platform Deep Dive/Scans/sequence-recorder.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Sequence recorder
```
Comment (mahmoudadly-cobalt) [inline content/en/Platform Deep Dive/Scans/sequence-recorder.md:1]:
```suggestion
# Sequence Recorder
```

File: content/en/Platform Deep Dive/Scans/sequence-recorder.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+Cobalt's Sequence Recorder is a browser plugin that records your actions in the browser. This can specifically
+help you with setting up an authenticated target. These actions will then be
+replayed by the scans when trying to authenticate to the target.
```
Comment (mahmoudadly-cobalt) [inline content/en/Platform Deep Dive/Scans/sequence-recorder.md:5]:
```suggestion
replayed by the DAST Scanner when trying to authenticate to the target.
```

File: content/en/Platform Deep Dive/Scans/sequence-recorder.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+# Installation
+
+_Work in progress_
```
Comment (mahmoudadly-cobalt) [inline content/en/Platform Deep Dive/Scans/sequence-recorder.md:9]: So we shouldn't merge yet?

File: content/en/Platform Deep Dive/Scans/sequence-recorder.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+# Usage
+
+Once installed, click on Cobalt's extension icon, a small window will pop up. Type in there
```
Comment (mahmoudadly-cobalt) [inline content/en/Platform Deep Dive/Scans/sequence-recorder.md:13]:
```suggestion
Once installed, click on Cobalt's extension icon, and a small window will pop up. Type in there
```

File: content/en/Platform Deep Dive/Scans/sequence-recorder.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+the URL of the target that you'd like to add authentication to and click `Start Recording`
```
Comment (mahmoudadly-cobalt) [inline content/en/Platform Deep Dive/Scans/sequence-recorder.md:14]:
```suggestion
the URL of the target that you'd like to add authentication to and click `Start Recording`.
```

File: content/en/Platform Deep Dive/Scans/sequence-recorder.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+
+![Click Start recording](/deepdive/scans/sequence-recorder/recorder1.png "Start Recording")
+
+You'll see a red dot (🔴) in the tab title, meaning that it's recording. Perform the necessary
```
Comment (mahmoudadly-cobalt) [inline content/en/Platform Deep Dive/Scans/sequence-recorder.md:19]: I believe it is a circle. At least it is the name of this icon.
```suggestion
You'll see a red circle (🔴) in the tab title, meaning that it's recording. Perform the necessary
```

File: content/en/Platform Deep Dive/Scans/sequence-recorder.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+actions to authenticate. Once done, click on the extension icon again and click `Stop Recording`.
+
+![Click Stop recording](/deepdive/scans/sequence-recorder/recorder2.png "Stop Recording")
+
+A new tab will appear. You can either copy the sequence to the clipboard or download it as a file.
+
+![Get the sequence](/deepdive/scans/sequence-recorder/recorder3.png "Get the sequence")
+
+You can now go to your target settings on Cobalt and paste or upload the sequence there.
```
Comment (mahmoudadly-cobalt) [inline content/en/Platform Deep Dive/Scans/sequence-recorder.md:28]: App? Platform? 🤔 

```suggestion
You can now go to your target settings on Cobalt's platform and paste or upload the sequence there.
```

File: content/en/Platform Deep Dive/Scans/sequence-recorder.md [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+
+Once those steps are completed, scans against this target will try to authenticate using these steps.
```
Comment (mahmoudadly-cobalt) [inline content/en/Platform Deep Dive/Scans/sequence-recorder.md:30]: Not 100% sure about this one. I thought why use `those` and `these` for the same steps.
```suggestion
Once these steps are completed, scans against this target will try to authenticate using these steps.
```

File: content/en/Platform Deep Dive/Scans/sequence-recorder.md (hunk 3 above)
Comment (davidgm0) [reply content/en/Platform Deep Dive/Scans/sequence-recorder.md:9]: I want to merge it, these steps well come once we have the app published. It's a bit of a chicken and egg problem 🐣

**Review by mahmoudadly-cobalt — APPROVED** [review]

**Pull request merged**